### PR TITLE
docs: clarify log storage is not in operator

### LINKED ISSUE #290: Documentation: clarify that long-term log storage is outside the purview of operator
Long-term log retention needs to be handled as part of the
kubernetes cluster configuration, and is not part of CNPG

### PR DESCRIPTION
**Pull request by jsilvela**
Add clarity that log storage is not in the purview of cnpg.

Closes #290 

Signed-off-by: Jaime Silvela <jaime.silvela@enterprisedb.com>

### CODE REVIEW

**Review by phisco — APPROVED** [review]

File: docs/src/postgresql_conf.md [hunk 1]
```diff
@@ -317,6 +317,15 @@ host all all all scram-sha-256 # (or md5 for PostgreSQL version <= 13)
 
 Refer to the PostgreSQL documentation for [more information on `pg_hba.conf`](https://www.postgresql.org/docs/current/auth-pg-hba-conf.html).
 
+Inside the cluster manifest, `pg_hba` lines are added as list items
+in `spec.postgresql.pg_hba`. e.g.:
```
Comment (gbartolini) [inline docs/src/postgresql_conf.md:321]:
```suggestion
Inside the cluster manifest, `pg_hba` lines are added as list items
in `spec.postgresql.pg_hba`, as in the following excerpt:
```

File: docs/src/postgresql_conf.md [hunk 2]
```diff
@@ -317,6 +317,15 @@ host all all all scram-sha-256 # (or md5 for PostgreSQL version <= 13)
...
+
+``` yaml
+  postgresql:
+    pg_hba:
+      - host all all 10.244.0.0/16 md5
```
Comment (gbartolini) [inline docs/src/postgresql_conf.md:326]:
```suggestion
      - hostssl app app 10.244.0.0/16 md5
```

Comment (gbartolini) [reply docs/src/postgresql_conf.md:326]: I would rather encourage usage of SSL in examples. Also I am restricting access to the app database for the app user, using md5.

**Review by gbartolini — APPROVED** [review]

**Review by sxd — APPROVED** [review]

**Pull request merged**